project/ws/app/src/lib/routes/app-toc/components/app-toc-home/app-toc-home.component.html

```diff
@@ -357,7 +357,7 @@ <h3 class="margin-remove-bottom pt-4 info-msg" i18n *ngIf="content?.status === '
             <span class="certificate-text block">If the issue persists, please mail us at <a>mission[dot]karmayogi[at]gov[dot]in</a></span>
           </ng-container>
           <ng-container *ngIf="certData">
-            <a class="certificate_box" (click)="openCertificateDialog()">
+            <a class="certificate_box" (click)="openCertificateDialog(); raiseCertIntreactTelemetry()">
```

Comment:
Remove the semicolon else the build will fail.

Reply:
I've executed the prod build command locally without facing any problems, and we can use a semi-colon to separate function calls when necessary for multiple calls.